index.js

```diff
@@ -428,9 +428,10 @@ function createPanZoom(domElement, options) {
     }
   }
 
-  function handleSignleFingerTouch(e) {
-    e.stopPropagation()
-    e.preventDefault()
+  function handleSingleFingerTouch(e) {
+    // allow single touch propagation to enable link or image maps touching
+    // e.stopPropagation()
```

Comment:
These lines  were here for a reason. We can't simply comment them out, as that would break consumers who don't want touch events be propagated, without giving them a way out.

I'd suggest to pass a callback into `options` object, and if it is present, invoke the callback here with `e` as an argument. Then it would be up to that callback to invoke `e.stopPropagation()`

Makes sense?

Reply:
I needed to remove them to enable image maps areas touch. The callback solution makes sense for sure. I will try it out.

BTW, is there documentation for the library somewhere else than here? Yesterday I couldn't find documentation about options and bounds.

Reply:
The only documentation available is in the [README.md](https://github.com/anvaka/panzoom) file. Please feel free to extend it, and I'll do the same.